src/main/proto/openrtb-xapi.proto

```diff
@@ -0,0 +1,565 @@
+//
```

Comment:
This should not be a part of repository. As proto files are downloaded as part of build process. Please remove.